projects/angular/src/wizard/wizard.html

```diff
@@ -14,15 +14,18 @@
   [clrModalLabelledById]="wizardId"
 >
   <nav class="modal-nav clr-wizard-stepnav-wrapper">
-    <h1 class="clr-wizard-title" id="{{wizardId}}"><ng-content select="clr-wizard-title"></ng-content></h1>
+    <div class="clr-wizard-title" role="heading" [attr.aria-level]="wizardTitle.headingLevel" id="{{wizardId}}">
+      <ng-content select="clr-wizard-title"></ng-content>
+    </div>
     <clr-wizard-stepnav></clr-wizard-stepnav>
   </nav>
 
-  <h2 class="modal-title">
-    <span tabindex="-1" #wizardTitle class="modal-title-text">
-      <ng-template [ngTemplateOutlet]="navService.currentPageTitle"></ng-template>
-    </span>
-
+  <div class="modal-title">
```

Comment:
I think this change introduces visual changes. I would keep the heading element here for this change. There should not be any visual changes here. This PR should be scoped to just the custom heading levels. I have WIP on the wizard redesign that moves that header actions.